app/backend/error.py

```diff
@@ -22,6 +32,6 @@ def error_dict(error: Exception) -> dict:
 
 def error_response(error: Exception, route: str, status_code: int = 500):
     logging.exception("Exception in %s: %s", route, error)
-    if isinstance(error, APIError) and error.code == "content_filter":
+    if (isinstance(error, APIError) or isinstance(error, PromptProtectionError)) and error.code == "content_filter":
```

Comment:
Check out this cool syntactic sugar for what you're doing there (passing a tuple of types to check against)!
```suggestion
    if isinstance(error, (APIError, PromptProtectionError)) and error.code == "content_filter":
```

Reply:
Done